### DISCUSSION TITLE
FlexInspect support for external libraries

### DISCUSSION THREAD

**Discussion by scottdfedorov** [JetBrains/Qodana#278] [Q&A] [answered]
Am trying to get FlexInspect to work with external libraries, but running into blocks.

I have tried both using the `@file:DependsOn("com.google.guava:guava:32.1.2-jre")` at the top of the file and also 
```kotlin
val resolver = MainKtsResolvers.RepositoryResolver(
    repositories = listOf("https://repo.maven.apache.org/maven2/"),
    additionalDependencies = listOf("com.google.guava:guava:32.1.2-jre")
)
```
but am unable to add an external dependency. Does Qodana not support external dependencies for FlexInspect?

**Comment by brichbash** [comment, marked as answer]
It currently does not. There is [an issue](https://youtrack.jetbrains.com/issue/QD-10603/Support-external-libraries-in-FlexInspect) on our tracker for it, which I can see you already have voted for. Thanks for bringing this up.
